Library/Homebrew/linkage_checker.rb

```diff
@@ -92,6 +92,13 @@ def dylib_to_dep(dylib)
     Regexp.last_match(2)
   end
 
+  sig { params(file: String).returns(T::Boolean) }
+  def broken_dylibs_allowed?(file)
+    return false if formula.name != "julia"
```

Comment:
I noticed there is some logic afterward that checks `if formula`. I guess there is a chance formula will not be resolved?
https://github.com/Homebrew/brew/blob/5b0a84bc7a67da90964ba88377b7192e8482c893/Library/Homebrew/linkage_checker.rb#L317-L320

If so, can modify to check `if formula.blank? && ...`

Reply:
Could just do
```ruby
if formula&.name != "julia"
```